XCTestPlans/AWSIntegrationTestsOnCI.xctestplan

```diff
@@ -0,0 +1,95 @@
+{
+  "configurations" : [
+    {
+      "id" : "FF4C1A8F-5D70-4A2A-BFDE-77467500C547",
+      "name" : "Test Scheme Action",
+      "options" : {
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "environmentVariableEntries" : [
+
+    ]
+  },
+  "testTargets" : [
```

Comment:
This section of the file contains all the test targets to be run as part of integration tests.

Note that when we add a new integration test target to the repo, we will have to add it here to make it run on CI.